packages/credential-provider-imds/src/fromInstanceMetadata.ts

```diff
@@ -154,23 +200,113 @@ const getMetadataToken = async (options: RequestOptions) =>
     },
   });
 
-const getProfile = async (options: RequestOptions) => (await httpRequest({ ...options, path: IMDS_PATH })).toString();
+/**
+ * @internal
+ * Checks if IMDS credential fetching is disabled through configuration
```

Comment:
description must be the first part of the tsdoc block, above the `@internal` access level annotation